modules/gpgsqlbackend/schema.pgsql.sql

```diff
@@ -5,9 +5,9 @@ CREATE TABLE domains (
   last_check            INT DEFAULT NULL,
   type                  VARCHAR(8) NOT NULL,
```

Comment:
modules/gpgsqlbackend/4.3.0_to_4.7.0_schema.pgsql.sql converts `type` to TEXT and adds the new columns as TEXT, too. this is now inconsistent here